docs/source/top_level_elements/relatedItem.rst

```diff
@@ -0,0 +1,598 @@
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Digital Collection" type="host">
+      <titleInfo>
+        <title>Children's Defense Fund</title>
+      </titleInfo>
+    </relatedItem>
+
+
+
+relatedItem[@type = "host"][@displayLabel = "Project Part"]
+-----------------------------------------------------------
+
+This XPath is used 2632 times in the Arrowmont Collection; e.g. `arrow:1 <https://digital.lib.utk.edu/collections/islandora/object/arrow:1/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>From Pi Beta Phi to Arrowmont</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Project Part" type="host">
+      <titleInfo>
+        <title>The Arrow of Pi Beta Phi</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Bibliographic Citation" type="host">
+      <titleInfo>
+        <title>The Arrow, Volume 27, Number 1</title>
+      </titleInfo>
+    </relatedItem>
+
+The relationship expressed here in :code:`[@type='host'][@displayLabel='Project Part']` is structural in nature and should be reflected by default behavior in our DAMS.
+
+relatedItem[@type = "host"][@displayLabel = "Bibliographic Citation"]
+---------------------------------------------------------------------
+
+This XPath, closely related to the preceding :code:`[@displayLabel = "Project Part"]`, also only appears 1264 times in the Arrowmont Collection - and only in the Arrow of Pi Beta Phi subcollection.
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:bibliographicCitation "The Arrow, Volume 27, Number 1" .
+
+relatedItem[@type = "host"][@displayLabel = "Is Part Of"]
+---------------------------------------------------------
+
+This XPath is only used 449 in the Volunteer Voices collection; e.g. `volvoices:1846 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:1846/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>Volunteer Voices</title>
+      </titleInfo>
+      <location>
+        <url>http://digital.lib.utk.edu/collections/volvoices</url>
+      </location>
+    </relatedItem>
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>Prints Collection</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Is Part Of" type="host">
+      <titleInfo>
+        <title>Harper's Weekly</title>
+      </titleInfo>
+    </relatedItem>
+
+:code:`[@displayLabel='Is Part Of']` may be sufficiently captured in other parts of the MODS (e.g. :code:`abstract`). Flip-flopping backwards, we should ignore this XPath.
```

Comment:
Agree 100%. For clarity I wondered if it would be helpful to include the abstract in the XML code block above.

Reply:
@mlhale7 done! Good suggestion - thank you.